docsrc/sasl/authentication_mechanisms.rst

```diff
@@ -52,30 +70,50 @@ Not sure how to get GSSAPI going? Check out our :ref:`GSSAPI configuration guide
 GSS-SPEGNO
 ----------
 
-.. todo::
-   Content needed here
+This is a Microsoft specific customization of GSSAPI.
+
+Described in the `Microsoft documentation <https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-adts/e1cbe214-d73b-4c58-aad2-bee399ccdfb8>`_ and :rfc:`4178`
 
 KERBEROS_V4
 -----------
 
-.. todo::
-   Content needed here
+Kerberos is/was used within Microsoft Active Directory.
```

Comment:
No, M$ never used Kerberos V4, it was already obsolete before M$ started supporting Kerberos.

V4 is just "historic" - hasn't been used by anyone in the 21st century.

Reply:
Ah, thanks for clarifying